setup.py

```diff
@@ -1,5 +1,18 @@
 from pathlib    import Path
 from setuptools import setup
+import sys
+
+min_version = (3, 6)
+
+if sys.version_info < min_version:
+    error = """
+Beginning with Matplotlib 3.1, Python {0} or above is required.
```

Comment:
Should this say "Beginning with augur 6.5.0, ..."?

Reply:
Whoops sorry about that!